linux-build/appimage/ayab.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Name=ayab
+Exec=ayab
+Comment=All Yarns Are Beautiful
+Icon=ayab
+Categories=Graphics;
+Terminal=false
+X-AppImage-Version=v0.95-104-gc97f0b6
```

Comment:
I guess we should update this field dynamically, as we do with `package_version`...time for another script? :D

Reply:
I think this field is updated automatically.